### PR TITLE
Translate project index page to Japanese

### LINKED ISSUE #46: [project-index Page] Translate to Japanese
## About the post

- [ ] I would like to suggest a topic;
- [x] I would like to write the post/article;

## Category

- [ ] News
- [ ] HTML / Semantic
- [ ] ARIA
- [ ] WCAG
- [ ] Tools / Testing
- [ ] Vue A11y packages / UI libraries
- [ ] Single page app

> None of the above categories? Describe a category that the subject of "post / article" makes the most sense.

- [x] Translate

## Brief description
Translate `jp/project/index.md`  to Japanese.

### PR DESCRIPTION
**Pull request by manak1**
I translated `jp/project/index.md` to Japanese.

close #46

### CODE REVIEW

**Comment by manak1** [comment]
@ktquez @yamanoku 

Please review this pull request when you guys have time.

File: src/jp/project/index.md [hunk 1]
```diff
@@ -1,21 +1,33 @@
...
+    content: 'https://vue-a11y.com/img/social-share-jp.png'
+  - name: 'twitter:title'
+    content: プロジェクト
+  - name: 'twitter:description'
+    content: Vue A11yは、Vue.jsに最大限のアクセシビリティをもたらす事を目的としたオープンソースプロジェクトです。
+  - name: 'twitter:image'
+    content: 'https://vue-a11y.com/img/social-share-jp.png'
 ---
 
 # Introduction
```
Comment (yamanoku) [inline src/jp/project/index.md:16]: Please translation!
How about `はじめに` ?

File: src/jp/project/index.md [hunk 2]
```diff
@@ -1,21 +1,33 @@
 ---
-description: Vue A11y is an open-source community project, where the main objective is to create content focused on Vue.js and bringing maximum accessibility
+title: プロジェクト
+description: Vue A11yは、Vue.jsに最大限のアクセシビリティをもたらす事を目的としたオープンソースプロジェクトです。
+
+meta:
+  - property: 'og:image'
+    content: 'https://vue-a11y.com/img/social-share-jp.png'
+  - name: 'twitter:title'
+    content: プロジェクト
```
Comment (yamanoku) [inline src/jp/project/index.md:9]:
```suggestion
title: はじめに
description: Vue A11yは、Vue.jsに最大限のアクセシビリティをもたらす事を目的としたオープンソースプロジェクトです。

meta:
  - property: 'og:image'
    content: 'https://vue-a11y.com/img/social-share-jp.png'
  - name: 'twitter:title'
    content: はじめに
```

The title should match the main heading.
https://github.com/vue-a11y/vue-a11y.com/pull/47/files#r563237561

File: src/jp/project/index.md [hunk 3]
```diff
@@ -1,21 +1,33 @@
...
 # Introduction
 
-**Vue A11y** is an open-source community project, where the main objective is to create content focused on Vue.js and bringing maximum accessibility.
+**Vue A11y**は、Vue.jsに最大限のアクセシビリティをもたらす事を目的としたオープンソースプロジェクトです。
 
-The project is open, so anyone who wants to participate is very welcome.
+プロジェクトはオープンなため、どなたでも貢献する事ができます。
 
-Accessibility is super important! We believe that if we join efforts to centralize content that is easy to understand, through recipes, tips, and blog posts, we can help developers and creators of Vue.js libraries to have consistent and more inclusive code.
+アクセシビリティはとても大切です！わかりやすいコンテンツを一元化させる事ができれば、レシピ、ヒント、そしてブログ記事を通し、開発者、Vue.jsライブラリーの開発者がより一貫性のある内包的なコードを書く事に貢献する事ができると信じています。
```
Comment (yamanoku) [inline src/jp/project/index.md:22]:
```suggestion
アクセシビリティはとても大切です！わかりやすいコンテンツの一元化に取り組むことで、レシピ、ヒント、そしてブログ記事を通じ、開発者や Vue.js ライブラリ制作者がより一貫性のある包括的なコードを書く事に貢献できると信じています。
```

File: src/jp/project/index.md [hunk 4]
```diff
@@ -1,21 +1,33 @@
...
+  - name: 'twitter:description'
+    content: Vue A11yは、Vue.jsに最大限のアクセシビリティをもたらす事を目的としたオープンソースプロジェクトです。
+  - name: 'twitter:image'
+    content: 'https://vue-a11y.com/img/social-share-jp.png'
 ---
 
 # Introduction
 
-**Vue A11y** is an open-source community project, where the main objective is to create content focused on Vue.js and bringing maximum accessibility.
+**Vue A11y**は、Vue.jsに最大限のアクセシビリティをもたらす事を目的としたオープンソースプロジェクトです。
```
Comment (yamanoku) [inline src/jp/project/index.md:18]:
```suggestion
**Vue A11y** は、Vue.js に最大限のアクセシビリティをもたらす事を目的としたオープンソースプロジェクトです。
```

[nits] white space

File: src/jp/project/index.md [hunk 5]
```diff
@@ -1,21 +1,33 @@
...
-Feel invited to **[contribute](/project/how-to-contribute.html)** in the development, in the creation of posts, recipes and tips, or by giving an incentive **[supporting](/project/how-to-contribute.html#supporting)** the project.
+記事、レシピ、ヒントを作成またはプロジェクトを **[支援](/jp/project/how-to-contribute.html#supporting)** する事で気軽に開発に **[貢献](/jp/project/how-to-contribute.html)** してください。
 
 ---
 
 <br>
 
 ::: alert Updates  
-We are just getting started. We will always keep you updated through [issues on GitHub](https://github.com/vue-a11y/vue-a11y.com/issues) and our [Twitter account](https://twitter.com/vue_a11y).  
+私たちはまだ始まったばかりです。何か更新があれば随時[GitHubのissue](https://github.com/vue-a11y/vue-a11y.com/issues)、そして[Twitter](https://twitter.com/vue_a11y)を通して連絡します。
```
Comment (yamanoku) [inline src/jp/project/index.md:31]:
```suggestion
私たちはまだ始まったばかりです。[GitHub の issue](https://github.com/vue-a11y/vue-a11y.com/issues) や、[Twitterアカウント](https://twitter.com/vue_a11y) を通していつでも最新情報をお知らせします。
```

File: src/jp/project/index.md [hunk 6]
```diff
@@ -1,21 +1,33 @@
 ---
-description: Vue A11y is an open-source community project, where the main objective is to create content focused on Vue.js and bringing maximum accessibility
+title: プロジェクト
+description: Vue A11yは、Vue.jsに最大限のアクセシビリティをもたらす事を目的としたオープンソースプロジェクトです。
```
Comment (yamanoku) [inline src/jp/project/index.md:3]:
```suggestion
description: Vue A11y は、Vue.js に最大限のアクセシビリティをもたらす事を目的としたオープンソースプロジェクトです。
```

[nits] white space

File: src/jp/project/index.md [hunk 7]
```diff
@@ -1,21 +1,33 @@
...
+
+meta:
+  - property: 'og:image'
+    content: 'https://vue-a11y.com/img/social-share-jp.png'
+  - name: 'twitter:title'
+    content: プロジェクト
+  - name: 'twitter:description'
+    content: Vue A11yは、Vue.jsに最大限のアクセシビリティをもたらす事を目的としたオープンソースプロジェクトです。
```
Comment (yamanoku) [inline src/jp/project/index.md:11]:
```suggestion
    content: Vue A11y は、Vue.js に最大限のアクセシビリティをもたらす事を目的としたオープンソースプロジェクトです。
```

[nits] white space

**Review by yamanoku — COMMENTED** [review]
@manak1 Thx, good contribution! I reviewed it!

**Comment by manak1** [comment]
@yamanoku 
Thank you for reviewing my pull request! 
I fixed it so, would you re review  the change?

File: src/jp/project/index.md [hunk 8]
```diff
@@ -1,21 +1,33 @@
...
+title: はじめに
+description: Vue A11y は、Vue.js に最大限のアクセシビリティをもたらす事を目的としたオープンソースプロジェクトです。
+
+meta:
+  - property: 'og:image'
+    content: 'https://vue-a11y.com/img/social-share-jp.png'
+  - name: 'twitter:title'
+    content: はじめに
+  - name: 'twitter:description'
+    content: Vue A11y は、Vue.js に最大限のアクセシビリティをもたらす事を目的としたオープンソースプロジェクトです。ジェクトです。
```
Comment (yamanoku) [inline src/jp/project/index.md:11]:
```suggestion
    content: Vue A11y は、Vue.js に最大限のアクセシビリティをもたらす事を目的としたオープンソースプロジェクトです。
```

[nits] Unnecessary words

**Review by yamanoku — COMMENTED** [review]
@manak1 thanks, fix document!
Please check https://github.com/vue-a11y/vue-a11y.com/pull/47/files#r563260826

**Comment by manak1** [comment]
@yamanoku 

Thank you for noticing! I'm sorry I should have double checked before requesting an review.

**Review by yamanoku — APPROVED** [review]
LGTM!
Thanks, good translation!

**Pull request merged**